pyfr/plugins/turbulence.py

```diff
@@ -0,0 +1,255 @@
+import math
+import random
+import time
+import uuid
+
+import numpy as np
+
+from collections import defaultdict
+from pyfr.plugins.base import BaseSolverPlugin
+from pyfr.regions import BoxRegion
+from rtree.index import Index, Property
+
+
+class pcg32rxs_m_xs:
```

Comment:
Rewrite this as a generator and inline the constants. @WillTrojak can advise.

Reply:
What is the advantage of using a generator? I liked the fact that with the class you had can have a get_state method that just gets the state without cycling the generator.

Reply:
So you can have a generator which evolves the state and then a function which goes from state to a float.